infrastructure_files/setup.env.example

```diff
@@ -7,6 +7,8 @@ NETBIRD_AUTH_OIDC_CONFIGURATION_ENDPOINT=""
 NETBIRD_AUTH_AUDIENCE=""
 # e.g. netbird-client
 NETBIRD_AUTH_CLIENT_ID=""
+# if you want to use a custom claim for the user ID instead of 'sub', set it here
+# NETBIRD_AUTH_USER_ID_CLAIM=""
```

Comment:
We could set the value to sub already here or on the base.setup.env

Reply:
As I mentioned, it set in the code if it is empty.